Resources/Prototypes/DeltaV/Entities/Structures/Doors/Windoors/windoor.yml

```diff
@@ -21,3 +21,27 @@
   components:
   - type: AccessReader
     access: [["Paramedic"]]
+
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecureChiefJusticeLocked
+  suffix: ChiefJustice, Locked
+  components:
+  - type: AccessReader
+    access: [["ChiefJustice"]]
+
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecureJusticeLocked
+  suffix: Justice, Locked
+  components:
+  - type: AccessReader
+    access: [["Justice"]]
+
+- type: entity
+  parent: WindoorSecure
+  id: WindoorSecureProsecutorLocked
+  suffix: Prosecutor, Locked
+  components:
+  - type: AccessReader
+    access: [["Prosecutor"]]
```

Comment:
Was going to mention this before but saw this again; does Prosecutor *need* their own access?

The button disappeared but this needs a line at the end.